README.md

```diff
@@ -4,6 +4,65 @@ This research repository maintains the code and the results for the research pap
 
 _"Set theory has become the standard foundation for mathematics, as every mathematical object can be viewed as a set."_ -[Stanford Encyclopedia of Philosophy](https://plato.stanford.edu/entries/set-theory/)
 
+# Install Package from PyPI and Use
+To install the package, please run:
+```pip install setlexsem```
+You can generate the dataset by:
```

Comment:
Is `the dataset` here a set of samples? Of prompts? Describe the inputs and outputs.

Reply:
sets. i'll update these all for clarity